javascript/search/binary_search.js

```diff
@@ -0,0 +1,19 @@
+// This function should be implemented, array is assumed to be sorted
+function binarySearch(array, searchItem) {
+    var present = array.indexOf(searchItem);
+    if (present == -1) {
+        throw "Item not present in array";
+    } else {
```

Comment:
Like I said in another comment, no need for an `else` branch here either. No big deal, but getting rid of it makes the code a bit cleaner.